docs/sidecar/comparison.md

```diff
@@ -0,0 +1,486 @@
+    }
+    
+    [HttpGet]
+    public async Task<ActionResult<User>> GetMe()
+    {
+        var user = await _downstreamApi.GetForUserAsync<User>(
+            "Graph",
+            options => options.RelativePath = "me"
+        );
+        return Ok(user);
+    }
+}
+```
+
+**After (Any language with HTTP client)**:
+
+C#:
+```csharp
+public class UserController : ControllerBase
+{
+    private readonly HttpClient _httpClient;
+    private readonly string _sidecarUrl;
+    
+    public UserController(IHttpClientFactory httpClientFactory, IConfiguration config)
+    {
+        _httpClient = httpClientFactory.CreateClient();
+        _sidecarUrl = config["SIDECAR_URL"];
+    }
+    
+    [HttpGet]
+    public async Task<ActionResult<User>> GetMe()
+    {
+        var token = Request.Headers["Authorization"].ToString();
+        var request = new HttpRequestMessage(
+            HttpMethod.Get,
+            $"{_sidecarUrl}/DownstreamApi/Graph?optionsOverride.RelativePath=me"
+        );
+        request.Headers.Add("Authorization", token);
+        
+        var response = await _httpClient.SendAsync(request);
+        var result = await response.Content.ReadFromJsonAsync<SidecarResponse>();
+        var user = JsonSerializer.Deserialize<User>(result.Content);
+        return Ok(user);
+    }
+}
+```
+
+TypeScript:
+```typescript
+export async function getMe(incomingToken: string): Promise<User> {
+  const sidecarUrl = process.env.SIDECAR_URL!;
+  
+  const response = await fetch(
+    `${sidecarUrl}/DownstreamApi/Graph?optionsOverride.RelativePath=me`,
+    {
+      headers: {
+        'Authorization': incomingToken
+      }
+    }
+  );
+  
+  const result = await response.json();
+  return JSON.parse(result.content) as User;
+}
+```
+
+#### Step 4: Remove Microsoft.Identity.Web Dependencies
+
+Remove NuGet packages from your project:
+
+```xml
+<!-- Remove these from .csproj -->
+<PackageReference Include="Microsoft.Identity.Web" Version="..." />
+<PackageReference Include="Microsoft.Identity.Web.MicrosoftGraph" Version="..." />
+<PackageReference Include="Microsoft.Identity.Web.DownstreamApi" Version="..." />
+```
+
+Remove authentication configuration:
+
+```csharp
+// Remove from Program.cs or Startup.cs
+services.AddMicrosoftIdentityWebApiAuthentication(Configuration)
+    .EnableTokenAcquisitionToCallDownstreamApi()
+    .AddDownstreamApi("Graph", Configuration.GetSection("DownstreamApis:Graph"))
+    .AddInMemoryTokenCaches();
+```
+
+If still validating tokens in your app, you can continue using authentication middleware, or delegate validation to the sidecar entirely.
+
+#### Step 5: Test and Validate
+
+1. **Unit Tests**: Update tests to mock HTTP calls to sidecar
+2. **Integration Tests**: Test sidecar communication in staging
+3. **Performance Tests**: Measure HTTP overhead impact
+4. **Security Tests**: Validate token handling and network policies
+
+### Migrating from Sidecar to Microsoft.Identity.Web
```

Comment:
This paragraph (Migrating from Sidecar to Microsoft.Identity.Web) is not needed.